readthedocs/projects/tests/test_views.py

```diff
@@ -0,0 +1,100 @@
+from django.contrib.auth.models import User
+from django.test import TestCase
+from django.urls import reverse
+from django_dynamic_fixture import get
+
+from readthedocs.integrations.models import Integration
+from readthedocs.projects.models import Project
+
+
+class TestExternalBuildOption(TestCase):
```

Comment:
Why are we testing this under `test_views.py`? Shouldn't we test this under `test_forms.py` since the logic change was on the form itself and does not depends on a view?

Reply:
I like testing most things at the view level, it covers more code instead of just the form.